tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1300,7 +1300,7 @@ public function testCreateIndexes(array $data): array
         ]);
 
         $this->assertEquals(400, $unknown['headers']['status-code']);
-        $this->assertEquals('Unknown attribute: Unknown', $unknown['body']['message']);
+        $this->assertStringContainsString('Unknown attribute: Unknown', $unknown['body']['message']);
```

Comment:
This appears to have been broken since https://github.com/appwrite/appwrite/pull/9105